<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advanced_graph_details</class>
 <widget class="QDialog" name="advanced_graph_details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced graph setting</string>
  </property>
  <widget class="QPushButton" name="but_ok">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_link_selection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Use selected links only</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="model_centroids">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>76</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="path_through_centroids">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Block path through connectors</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="set_centroids_rule">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Define number of centroids</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>use_link_selection</tabstop>
  <tabstop>path_through_centroids</tabstop>
  <tabstop>set_centroids_rule</tabstop>
  <tabstop>model_centroids</tabstop>
  <tabstop>but_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
